pyrasterframes/src/main/python/docs/raster-read.pymd

```diff
@@ -98,8 +101,10 @@ modis_catalog = spark.read \
 modis_catalog.printSchema()
 
 print("Available scenes: ", modis_catalog.count())
+```
 
-modis_catalog.show(5, truncate=False)
+```python, show_catalog, results='raw'
+modis_catalog.show(5, truncate=True)
```

Comment:
this one has some line wraps but is overall too wide, not responsive, no scroll bar